src/panels/lovelace/hui-root.ts

```diff
@@ -58,6 +58,8 @@ import type { Lovelace } from "./types";
 import "./views/hui-panel-view";
 import type { HUIPanelView } from "./views/hui-panel-view";
 import { HUIView } from "./views/hui-view";
+import { RequestSelectedDetail } from "@material/mwc-list/mwc-list-item";
```

Comment:
```suggestion
import type { RequestSelectedDetail } from "@material/mwc-list/mwc-list-item";
```

Reply:
If it is only a type it will be thrown out on compile, but I guess it can't hurt to add it.

Reply:
Oh it was duplicate import ofc 😄